src/admin/views/ConfigurationView.js

```diff
@@ -147,6 +148,119 @@ const LoadingTable = () => (
 );
 
 const ConfigurationView = () => {
+  var STATUS = gql`
```

Comment:
Do you need to have this definition of the status query? Instead of relying on refetchQueries, maybe  the status query can be inside a container component and rendered after the fragments are built so we don't have to make many different calls to the status query.